readme.md

```diff
@@ -23,22 +23,25 @@ the game, including new versions, custom maps and much more.
 
 Originally written by Magnus Auvinen.
 
+---
 
-Building on Linux or macOS
+Teeworlds supports two build systems: CMake (widely used) and bam (fast build times).

```

Comment:
This is still not true for me, I can only observe bam being faster if you forget to configure your `make`.
https://gist.github.com/Learath2/e62c57066859122b1f584c4d8ebaf157

bam does indeed have an edge while configuring, but again it configures much less so it is expected.

Reply:
That is the first time I hear about bam being slower than `make`. In all the experiments I did (with `-j`, of course) make was much slower. Ninja was slower on my usual machine (18s vs 13s), and similar on a high performance 12-core machine (3s vs 3s).

### Build
make -j4  65.99s user 4.10s system 368% cpu 19.038 total

ninja  65.54s user 3.26s system 378% cpu 18.189 total

bam conf=release  49.35s user 2.99s system 390% cpu 13.405 total

bam  31.62s user 3.78s system 382% cpu 9.245 total

### Config
cmake ..  2.13s user 0.78s system 101% cpu 2.875 total

bam config  0.10s user 0.00s system 100% cpu 0.104 total

Reply:
Office machine (8 core)

### Build
make -j8
real	0m9.049s
user	0m48.333s
sys	0m2.845s

ninja
real	0m8.676s
user	0m47.556s
sys	0m2.362s

bam conf=release
real	0m6.880s
user	0m43.153s
sys	0m2.496s

### Config
cmake ..
real	0m2.004s
user	0m1.441s
sys	0m0.568s

bam config
real	0m0.241s
user	0m0.169s
sys	0m0.193s

Reply:
Here's my low-end machine (Arch Linux, Intel(R) Celeron(R) CPU 1007U @ 1.50GHz):

Build
=====

ninja
Executed in   44.17 secs   fish           external 
   usr time   68.03 secs    0.23 millis   68.03 secs 
   sys time    7.99 secs    3.65 millis    7.99 secs 

bam
Executed in   46.82 secs   fish           external 
   usr time   70.65 secs    0.00 millis   70.65 secs 
   sys time   10.35 secs    4.32 millis   10.34 secs 


Config
======

cmake .. -DDEV=ON -GNinja
Executed in    6.83 secs   fish           external 
   usr time    4.87 secs  685.36 millis    4.19 secs 
   sys time    2.35 secs   24.05 millis    2.33 secs 

bam config
Executed in    1.02 secs   fish           external 
   usr time  982.01 millis    0.00 millis  982.01 millis 
   sys time  890.11 millis    5.06 millis  885.04 millis

Reply:
What OS are you building on, @Dune-jr?

Reply:
@heinrich5991 Linux Mint 19.3, cmake 3.10.2, GNU Make 4.1, ninja 1.8.2

Reply:
macOS 10.15, cmake 3.17.2, GNU Make 3.81, ninja 1.10.0 is what I ran this test on

Reply:
cmake 3.17.1, GNU Make 4.3, ninja 1.10.0, bam 0.5.1 using Lua 5.3.3 for me.